system-int-test/dev/user.clj

```diff
@@ -7,12 +7,9 @@
         [clojure.repl]
         [alex-and-georges.debug-repl]))
 
-
 (defn reset []
-
   (s/stop)
-
-  ; Refreshes all of the code in repl
+  ; Refreshes all the code in repl
```

Comment:
so this is a lint ticket so I have to say, two ;; for comments we are keeping.  Sorry.